app/src/main/res/layout/component_sessions.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.recyclerview.widget.RecyclerView
+        xmlns:android="http://schemas.android.com/apk/res/android" xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:id="@+id/session_list"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:layout_margin="16dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/toolbar_wrapper"/>
```

Comment:
Should this know what it's being constrained to? I think the owner of the component should be responsible for where it lays out.

Reply:
I agree. I was just testing having some default constraints. In the prototype, I used Anko to establish programmatic constraints for that fragment based on any layout experiments. I can make that change here.